contentful/content-migrations/migrations/20241014-1117-fix-releases-widget.js

```diff
@@ -0,0 +1,22 @@
+const Migration = require("contentful-migration");
+const { runForEachContentType } = require("../helpers");
+
+/**
+ * 
+ * @param {Migration} migration 
+ */
+module.exports = function (migration) {
+  runForEachContentType((contentTypeName) => updateContentType(migration, contentTypeName));
+};
+
+/**
+ * 
+ * @param {Migration} migration 
+ * @param {string} contentType 
+ */
+function updateContentType(migration, contentType) {
+  var contentType = migration.editContentType(contentType);
+
+  contentType.removeSidebarWidget("builtin", "releases-widget");
+  contentType.addSidebarWidget("sidebar-builtin", "releases-widget", undefined, "6RKxbgPghdY4llDpwCFvgR");
```

Comment:
what are last two params for?

Reply:
```
  /**
   * Adds a builtin or custom widget to the sidebar of the content type.
   *
   * @param widgetNamespace The namespace of the widget. Use 'sidebar-builtin' for standard widgets or 'extension' for UI extensions.
   * @param widgetId The ID of the builtin or extension widget to add.
   * @param settings Instance settings for the widget
   * @param insertBeforeWidgetId Insert widget above this widget in the sidebar. If null, the widget will be added to the end.
   */
  addSidebarWidget(
    widgetNamespace: 'sidebar-builtin' | 'extension' | 'app',
    widgetId: string,
    settings?: ISidebarWidgetSettings,
    insertBeforeWidgetId?: string | null
  ): void
```

Reply:
My last migration had the `widgetId` and `insertBeforeWidgetId` inversed, whoops.